### PR TITLE
Update govuk-frontend to v5.4.0

### PR DESCRIPTION
**Pull request by patrickjfl**
# Context

# Changes in this PR

### After

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: assets/scss/application.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ $path: '/assets/images/';
 $moj-page-width: 1170px;
 $govuk-page-width: $moj-page-width;
 
-@import 'govuk/all';
+@import 'govuk/index';
```
Comment (tahb) [inline assets/scss/application.scss:7]: It can be helpful to link directly to the release notes https://github.com/alphagov/govuk-frontend/releases/tag/v5.4.0 just to keep everything together and make it super simple for the reviewer.

File: assets/scss/application.scss [hunk 2]
```diff
@@ -30,7 +30,7 @@ $govuk-page-width: $moj-page-width;
 }
 
 .govuk-panel--warning {
-  background-color: #FAC710;
+  background-color: #fac710;
```
Comment (tahb) [inline assets/scss/application.scss:33]: A very minor change obviously but I don't _think_ this is needed as part of the upgrade? Guessing it was a style thing you noticed in passing? I'd put this in another commit so it's separate from the riskier change should that part need to be reverted for whatever reason, you're other work will persist 👍

File: assets/scss/application.scss (hunk 1 above)
Comment (tahb) [reply assets/scss/application.scss:7]: Did this work for you locally? I got an error on this line where index couldn't be found. When I changed it to the following (based on the release notes) it worked:

```
@import "govuk/base";
@import "govuk/core/index";
@import "govuk/objects/index";
@import "govuk/components/index";
@import "govuk/utilities/index";
@import "govuk/overrides/index";
```

File: assets/scss/application.scss (hunk 2 above)
Comment (patrickjfl) [reply assets/scss/application.scss:33]: hmmm very strange, I didn't change this at all. It must have happened automatically somehow. Anyway I will remove it, thanks

File: assets/scss/application.scss (hunk 1 above)
Comment (patrickjfl) [reply assets/scss/application.scss:7]: it did work for me locally, but let me try again.

Comment (patrickjfl) [reply assets/scss/application.scss:7]: OK now it's not. I thought I had deleted the stylesheets previously to reset the thing but I guess not. I will update this with your suggestion (which also works locally for me).

**Review by tahb — APPROVED** [review]
LGTM 👍
